### PR TITLE
LPS-97280 Extract asset list query logic out from ServiceBuilder model implementation

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

File: modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/internal/util/AssetListHelperImpl.java [hunk 1]
```diff
@@ -444,6 +444,9 @@ private long _getFirstSegmentsEntryId(
 
 				return assetListEntrySegmentsEntryRel != null;
 			}
+		).filter(
+			segmentsEntryId ->
+				segmentsEntryId != SegmentsConstants.SEGMENTS_ENTRY_ID_DEFAULT
```
Comment (dgarciasarai) [inline modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/internal/util/AssetListHelperImpl.java]: review this

**Comment by dgarciasarai** [comment]
https://github.com/dgarciasarai/liferay-portal

**Comment by epgarcia** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by epgarcia** [comment]
Pull request submitted to https://github.com/dgarciasarai/liferay-portal/pull/71. [See changes here.](https://github.com/epgarcia/liferay-portal/compare/d902196f2c32~3...d902196f2c32) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**